src/main/java/be/polyscripts/contactmanagerapp/exceptions/exceptionResolver/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package be.polyscripts.contactmanagerapp.exceptions.exceptionResolver;
+
+import be.polyscripts.contactmanagerapp.exceptions.FreelanceWithNoTVAException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice

```

Comment:
i give you a star here for exception handling 🤩

Reply:
Thanks ! 😊